packages/react-composites/src/composites/CallComposite/components/Banner.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Theme, mergeStyles } from '@fluentui/react';
+import { _pxToRem } from '@internal/acs-ui-common';
+// eslint-disable-next-line no-restricted-imports
+import { IIconProps, Icon, PrimaryButton, Stack, Text, useTheme } from '@fluentui/react';
+import React from 'react';
+
+/**
+ * Props for {@link Banner}.
+ *
+ * @private
+ */
+export interface BannerProps {
+  /**
+   * Banner strings.
+   */
+  strings?: BannerStrings;
+
+  /**
+   * Banner icon.
+   */
+  iconProps?: IIconProps;
+
+  /**
+   * Callback called when the primary button inside banner is clicked.
+   */
+  onClickPrimaryButton?: () => void;
+}
+
+/**
+ * All strings that may be shown on the UI in the {@link Banner}.
+ *
+ * @private
+ */
+export interface BannerStrings {
+  /**
+   * Banner title.
+   */
+  title: string;
+  /**
+   * Banner primary button label.
+   */
+  primaryButtonLabel: string;
+}
+
+/**
+ * A component to show a banner in the UI.
+ *
+ * @private
+ */
+export const Banner = (props: BannerProps): JSX.Element => {
```

Comment:
Do we need a new component? or is this something that Caroline's new notification component could be used for?

Reply:
It is very similar to here component but there is: 
1. no dismiss button
2. the button is on the same line as the title
It is an internal composite component for now but I make changes to new notification component later on.

Reply:
This makes sense, I thought there might be a mobile formfactor or something that we could use but yes I recall the buttons only render below the text